src/fmtargs.h

```diff
@@ -44,9 +44,20 @@
 /* Everything below this line is automatically generated by
  * generate-fmtargs.py. Do not manually edit. */
```

Comment:
This code is automatically generated, we need to fix the generator too.

Reply:
Hehe, we can make the generator emit /* clang-format off */.

Reply:
added fmtargs.h to .clang-format-ignore